src/containers/HomeContainer.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import Home from ".../components/home";
+
+class HomeContainer extends React.Component{
```

Comment:
No need to use the class version of react, if you're not manipulating the state just use the functional style components such as: 

`const HomeContainer = props =>(<Home/>)`

Then just export the HomeContainer